components/AboutTabs.tsx

```diff
@@ -22,13 +22,13 @@ function Members() {
 }
 
 function Alumni() {
```

Comment:
Can we move the TEST people to the bottom of the alumni page instead of the top? When we have it separated by TEST and grad years later on, we can do it as such:

```
Class of 2023
...

TEST Class of 2023
...

Class of 2022
...

TEST Class of 2022
...
```